static/app/data/platformCategories.tsx

```diff
@@ -313,33 +342,27 @@ export const replayPlatforms: readonly PlatformKey[] = [
   'javascript',
 ];
 
+// These are all the platforms that can set up replay.
+export const replayPlatforms: readonly PlatformKey[] = [
+  ...replayFrontendPlatforms,
+  ...replayBackendPlatforms,
+];
+
```

Comment:
i like it. we can export the fe or be lists in the future if we need to get specific too. 👍